### PR TITLE
[htop] Update to 3.0.2. Cleanup spec. JB#50718

### PR DESCRIPTION
**Pull request by mlehtima**

### CODE REVIEW

**Review by mkosola — APPROVED** [review]

File: rpm/htop.spec [hunk 1]
```diff
@@ -1,16 +1,15 @@
 Name:       htop
 Summary:    Interactive process viewer
-Version:    2.0.2
+Version:    3.0.2
 Release:    1
-Group:      System/Binaries
 License:    GPL 2.0
```
Comment (saukko) [inline rpm/htop.spec]: This should be GPLv2 (or +?)

Comment (saukko) [reply rpm/htop.spec]: Also %doc > %license in %files

**Review by saukko — APPROVED** [review]
Couple of comments but in general looks good to me.

File: rpm/htop.spec (hunk 1 above)
Comment (mlehtima) [reply rpm/htop.spec]: Fixed

**Review by jusa — APPROVED** [review]

**Pull request merged**